### PR TITLE
Admin web

### PR DESCRIPTION
**Pull request by b2simms**
Creating a CRUD interface using Material Design

### CODE REVIEW

File: src/main/java/org/AuthenticateDbHandler.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    	try {
+            stmt = (OraclePreparedStatement) con.prepareStatement(
+            		"select "
+            		+ "ID,"
+            		+ "NAME,"
+            		+ "COMPANIES_EMPLOYEE_ID,"
+            		+ "COMPANIES_ID,"
+            		+ "MANAGER,"
+            		+ "SUPER_ADMIN "
+            		+ "from employees where name = ? and web_password = ? ");
```
Comment (JoshNorthrup) [inline src/main/java/org/AuthenticateDbHandler.java:63]: we should change the `name` column to `companies_employee_id`

File: src/main/java/org/AuthenticateDbHandler.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+            stmt.setString(1, username);
+            stmt.setString(2, password);
+            ResultSet i = stmt.executeQuery();
+            
+            if(i.next()){
+	        	//parse result
+	            int iter = 1;
+	        	int id = Integer.parseInt(i.getString(iter++));
+	        	String name = i.getString(iter++);
+	        	int comp_emp_id = Integer.parseInt(i.getString(iter++));
```
Comment (JoshNorthrup) [inline src/main/java/org/AuthenticateDbHandler.java]: this assumes `companies_employee_id` is an int, we discussed making it a String

File: src/main/java/org/AuthenticateDbHandler.java (hunk 1 above)
Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:63]: Okay, if it is not used elsewhere this is an easy change

File: src/main/java/org/AuthenticateDbHandler.java (hunk 2 above)
Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java]: We can do that. I think my local database has it set to `int` rather that `varchar2 256`

File: src/main/java/org/AuthenticateDbHandler.java (hunk 1 above)
Comment (JoshNorthrup) [reply src/main/java/org/AuthenticateDbHandler.java:63]: The only other places I found were in the Employee class, the variable doesn't seem to be used anywhere else though

Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:63]: That is the case in my dev environment as well.
I can update the code to reflect the change from `int` to `varchar2 with length 256`

Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:63]: I just submitted it as an issue instead of changing it right now

**Review by JoshNorthrup — APPROVED** [review]
Now that I've got he auth this figured out everything on this branch seems to be working for me. I'm going to merge this. We'll discuss tomorrow but we should work on getting the code we've written in the past month or so up here and merged together so we can make sure everything is working together.

**Pull request merged**